lib/ansible/module_utils/network/nxos/argspec/interfaces/interfaces.py

```diff
@@ -47,7 +47,6 @@ def __init__(self, **kwargs):
                     'type': 'str'
                 },
                 'enabled': {
-                    'default': True,
```

Comment:
interfaces module is meant to have default being set for `enabled` parameter as True as it doesn't appear in running-config.

Reply:
I'll take another look at this. Legacy code ethernet defaulted to shutdown but it looks like the default may have changed to no shut somewhere after 9.2(2).

Reply:
@trishnaguha I've updated the code to handle the problems with enable and also updated the PR summary with more notes.